Design view files

lilpacy opened the pull request:
WHAT
index.hamlのマークアップ

lilpacy committed:
rails g controller comments

lilpacy committed:
make index.html.haml

lilpacy committed:
set scssfiles

alter application.css to ~.scss
set _reset.scss

lilpacy committed:
modification application.html.haml

lilpacy committed:
install font-awesome-rails

WHY
 ペンと歯車のアイコンを挿入するgemを入れる為

lilpacy committed:
Add description to  application.scss

WHY
 font-awesomeを使えるようにするための記述

lilpacy committed:
delete controller comments

WHY
 messageコントローラーに置き換える為

lilpacy committed:
generate controller messages

WHY
 messagesコントローラーを作る為

lilpacy committed:
alter routes.rb

WHY
 ルートパスを設定する為

lilpacy committed:
markup index

t-kusakabe commented on Gemfile:
```diff
@@ -26,6 +26,10 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Use haml at rails version 5.0.2
 gem 'haml-rails'
 
+# Add font-aweme
```
誤字ですかね。。。。

lilpacy replied:
誤字ですね！(>_<)
修正いたします！！

t-kusakabe commented on app/assets/javascripts/messages.coffee:
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
このファイルいらないなら消しておきましょ！

lilpacy replied:
了解しました！！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,194 @@
+  background-color: #38AEF0;
+  min-width: 980px;
+  color: #fff;
+  display: flex;
```
これ良いですね！

lilpacy replied:
結構気に入りました！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,194 @@
+    width: 300px;
+    height: 1000px;
+    background-color: #2f3e51;
+    color: #fff;
```
何度も追加そうな色とか高さは変数化してあげた方がいいかもです！

lilpacy replied:
はい！whiteで統一します！！

lilpacy replied:
統一終了しました！！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,194 @@
+            padding-top: 30px;
+            width: 202px;
+            font-size: 20px;
+            .group-name-main {
```
これなにしてますかね？

lilpacy replied:
消し忘れてました！(>_<)

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,194 @@
+  }
+}
+
+.clearfix:after {
```
mix-inしましょ!

lilpacy replied:
はい！今回はclearfix使わなかったので消しました！！

t-kusakabe commented on app/views/layouts/application.html.haml:
```diff
@@ -1,7 +1,7 @@
 %html
   %head
-    %meta{content: "text/html; charset=UTF-8", http-equiv: "Content-Type"}/
+    %meta{content: "text/html", charset: "UTF-8", http: {equiv: "Content-Type"}}
```
ハッシュスペースほしいです！

lilpacy replied:
スペース！！！

t-kusakabe replied:
`%meta { content: "text/html", charset: "UTF-8", http: { equiv: "Content-Type" }}`
こんなイメージです><

t-kusakabe commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,81 @@
+            %span
+              name
+          %span
+            %a{href: ""}
```
link_toつかいましょか！

lilpacy replied:
link_to！！！

t-kusakabe commented on test/controllers/messages_controller_test.rb:
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
いらないファイルは作らない方向で！

lilpacy replied:
消します！！

koutaodawara commented on app/assets/javascripts/comments.coffee:
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
このファイルいらないので消しちゃいましょう！

lilpacy replied:
はい！

koutaodawara commented on app/assets/stylesheets/comments.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the comments controller here.
```
これも使わないですね！

lilpacy replied:
はいい！

koutaodawara commented on app/helpers/comments_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```
これも消しましょう！

使うタイミングで生成していきましょう！

lilpacy replied:
はい！！

koutaodawara commented on test/controllers/comments_controller_test.rb:
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
rspecでテストを書くのでこのファイルも消しときましょう！

koutaodawara commented on app/helpers/messages_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module MessagesHelper
+end
```
消しましょう

lilpacy replied:
わかりました！！

lilpacy committed:
modification

WHAT
remove not necessary files
convert sentences to variables
correct spelling mistakes

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,193 @@
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+@mixin align-center() {
```
mix-in用のファイル作った方がいいかもです！

lilpacy replied:
はい！！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,193 @@
+  align-items: center;
+}
+
+$light-blue: #38aef0;
```
変数用のファイルを作った方がいいかもです！

lilpacy replied:
はいい！！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,193 @@
+          .user-name{
+            font-size: 16px;
+            font-weight: bold;
+            padding: 0 5px 0 0;
```
padding-rightとかではだめですか？

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,193 @@
+            color: #434a54;
+          }
+          .datetime {
+            padding: 0 0 0 5px;
```
こちらも下側の指定だけで！

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -0,0 +1,193 @@
+          }
+        }
+        .massage-body {
+          padding: 10px 0 40px 0;
```
最後の0は別に必要ないかもです！

lilpacy committed:
modification2

make mix-in and variable scssfiles
insert some spaces
correct padding syntax

t-kusakabe commented on app/assets/stylesheets/_messages.scss:
```diff
@@ -2,12 +2,8 @@
-}
-
-$light-blue: #38aef0;
+@import "mix-in";
```
このやり方だと、今後別ファイルでmix-inで定義したものをつかいた時に、もう一度呼び出す必要がでてきます。
現状問題ないですが、大半のファイルで読み込む必要が出てきた場合にはapplicationとかで読み込むようするといいかもです！

lilpacy replied:
application.scssに書き直しました！！

t-kusakabe commented:
良いかと！
あとは @koutaodawara 氏の判断を仰ぎましょうｗ

lilpacy committed:
modification3

WHAT
import mix-in and variable in application.scss

lilpacy committed:
modification4

WHAT
change import_place of mixin and variable files

koutaodawara commented:
whiteはめっちゃ使ってるので変数におきましょうか！！！！！

そこ直したらLGTMで１

lilpacy committed:
modification5

convert description_white into mixin
